geomesa-arrow/geomesa-arrow-gt/src/main/scala/org/locationtech/geomesa/arrow/io/DeltaWriter.scala

```diff
@@ -572,21 +599,30 @@ object DeltaWriter extends StrictLogging {
     // calculate our vector bindings/metadata once up front
     val vectorMetadata = dictionaryFields.toArray.map { name =>
       val descriptor = sft.getDescriptor(name)
-      val bindings = ObjectType.selectType(descriptor)
-      val metadata = Map(SimpleFeatureVector.DescriptorKey -> SimpleFeatureTypes.encodeDescriptor(sft, descriptor))
+      val bindings = {
+        val base = ObjectType.selectType(descriptor)
+        if (descriptor.isList) { base.tail } else { base }
+      }
+      val ordering = AttributeOrdering(bindings)
+      val desc = if (descriptor.isList) { s"$name:${descriptor.getListType().getSimpleName}" } else {
+        SimpleFeatureTypes.encodeDescriptor(sft, descriptor)
```

Comment:
Should this logic just be rolled into the SFTs.encodeDescriptor?  Or is that function too general for this specific, Arrow-related use?